Sources/FileSystem/FileSystem.swift

```diff
@@ -52,6 +52,24 @@ public enum MakeDirectoryOptions: String {
     case createTargetParentDirectories
 }
 
+/// Options to configure the writing of text files.
+public enum WriteTextOptions {
+    /// When passed, it overrides any existing files.
+    case overriding
```

Comment:
```suggestion
    case overwrite
```

Shouldn't this be `overwrite` rather than override? They sound similar, but I don't think you "override" a file, you overwrite it.